SampleReadme.md

```diff
@@ -0,0 +1,243 @@
+# Table of Contents
+[Concepts introduced in this sample](#concepts-introduced-in-this-sample)
+[Prerequisites](#prerequisites)
+[Configure Services](#configure-services)
+[Run the Sample](#run-this-sample)
+[Testing the bot using Bot Framework Emulator](#testing-the-bot-using-bot-framework-emulator)
+[Deploy to Azure](#deploy-to-azure)
+[Further Reading](#further-reading)
+
+# Concepts introduced in this sample
+<DESCRIPTION OF THE CONCEPTS>
+
+- Services used in this sample
+- [LUIS](https://luis.ai) for Natural Language Processing
+- [QnA Maker](https://qnamaker.ai) for FAQ, chit-chat, getting help and other single-turn conversations
+- [Dispatch](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-tutorial-dispatch?view=azure-bot-service-4.0&tabs=csharp) For multiple QnA and/or LUIS applications
+- [AppInsights](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-manage-analytics?view=azure-bot-service-4.0) For Telemetry in your Bot
+
+# Prerequisites
+
+###	Required Tools
+- [.NET Core SDK](https://www.microsoft.com/net/download/dotnet-core/2.1) version 2.1.403 or higher
+- [Node.js](https://www.microsoft.com/net/download/dotnet-core/2.1) version 8.5 or higher 
+- [Azure CLI tools](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) (Optional)
+- [Bot Builder Tools](https://github.com/Microsoft/BotBuilder-tools) (Optional)
+
+### Clone the repo
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+## Configure Services

```

Comment:
I think more context is required here.  Conceptually we spoke about 3 paths - using CLI/Portal and Clone.  Following this guide, i'm just going to dive in and begin the manual process of building a bot configuration without any concept of which adventure I chose.